projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -418,7 +418,7 @@
 
 		&[type="radio"]::before {
 			border-radius: 50%;
-			transform: translateY(15%);
+			transform: translateY(0.15em);
```

Comment:
Check 2017 theme. In the editor, the `transform: translateY(15%);` was causing a 3d effect. Frontend looks okay.

<img width="796" alt="Screenshot 2025-05-22 at 11 37 07 am" src="https://github.com/user-attachments/assets/e7f171d4-5571-4254-93d5-d14e4f2a3643" />

Reply:
I think the actual input element is supposed to be hidden by some css (`appearance: none`) in favour of a pseudoelement. Maybe there's something overriding that in some classic themes.

Reply:
This was actually due to incoming styles from the classic themes. The jetpack form sets the width/height to be `1em`, and adds borders and things.

Reply:
I think I found out why: https://github.com/Automattic/jetpack/pull/43531/files#r2101527871

I compared with trunk and it fixes the 3d effect. I believe it has to do with the change to `1em` sizing for the input field, also we removed the `border: none` override.

🤞🏻